pkg/k8s/network_policy.go

```diff
@@ -86,13 +73,24 @@ func parseNetworkPolicyPeer(namespace string, peer *networkingv1.NetworkPolicyPe
 			lsr.Key = policy.JoinPath(k8sConst.PodNamespaceMetaLabels, lsr.Key)
 			peer.NamespaceSelector.MatchExpressions[i] = lsr
 		}
-	} else {
-		// Neither PodSelector nor NamespaceSelector set.
-		return nil
+
+		selector := api.NewESFromK8sLabelSelector(labels.LabelSourceK8sKeyPrefix, labelSelector, peer.PodSelector)
+		retSel = &selector
```

Comment:
is the return from `NewESFromK8sLabelSelector` something we can't take the address of directly?

Reply:
correct :(